recipe/meta.yaml

```diff
@@ -36,16 +37,20 @@ test:
     - twine
     - twine.commands
   commands:
+    - twine --version
     - twine --help
 
 about:
   home: https://github.com/pypa/twine
-  license: Apache  2.0
+  license: Apache-2.0
   license_family: Apache
   license_file: LICENSE
   summary: 'Collection of utilities for interacting with PyPI'
+  doc_url: https://twine.readthedocs.io
+  doc_source_url: https://github.com/pypa/twine/tree/master/docs
 
 extra:
   recipe-maintainers:
     - jankatins
```

Comment:
I think it is time to say goodbye here: can you remove myself from the maintainer list?

Reply:
:crying_cat_face: thanks for all the packaging!

Reply:
removed. maybe drop a note down in the main body of the issue as well (code comments get hard to find sometimes)